server/src/acccount-manager/dto/auth.dto.ts

```diff
@@ -16,18 +19,30 @@ export class ResetPasswordDto {
 }
 
 export class ReturnUserDto {
+  @Expose()
   id: number;
+  @Expose()
   firstName: string;
-  last_name: string;
+  @Expose()
+  last_name?: string;
```

Comment:
Do you think it's worth to add a migration to update field names to follow a naming convention with camel-case instead of snake-case, or vice-versa?  I think my preference is camelCase as it aligns with the front-end, but I'm not sure what is more common with Node/Nest.js?

Reply:
That's a good point. This was done, I believe to make table column names conform to the Postgres standard, but there should be a way to tell Typeorm to use camel case in JS.